_health-care/_js/utils/options-for-select.js

```diff
@@ -237,6 +237,18 @@ const maritalStatuses = [
   'Divorced'
 ];
 
+const branchesServed = [
+  'Army',
+  'Airforce',
+  'Marine',
+  'Navy'
+];
+
+const dischargeTypes = [
```

Comment:
Good info to have, thanks Joe! Do we need to exactly match the options that the xml text has? E.g., is it okay if we submit `FILIPINO NEW SCOUT` instead of `F.SCOUTS NEW`? I'm not sure how all of this is processed on their end.